packages/ckeditor5-coremedia-richtext/src/rules/FixedAttributes.ts

```diff
@@ -0,0 +1,39 @@
+import { RuleConfig } from "@coremedia/ckeditor5-dom-converter";
+
+const fixedAttributes = new Map<string, Map<string, string>>([
+  ["pre", new Map([["xml:space", "preserve"]])],
+  ["a", new Map([["xlink:type", "simple"]])],
+  [
+    "img",
+    new Map([
+      ["xlink:type", "simple"],
+      ["xlink:show", "embed"],
+      ["xlink:actuate", "onLoad"],
+    ]),
+  ],
+]);
+
+/**
+ * RichTextSanitizer keeps an attribute with a valid fixed value.
+ * If such an attribute should be removed before storing, then this rule can be used.
+ * @example For an attribute with valid fixed value
+ * ```html
+ * <a href="" xlink:type="simple">
+ * ```
+ */
+export const stripFixedAttributes = (): RuleConfig => ({
```

Comment:
I bet you evaluated it in our example application, but: Do we need a priority here? The rule requires that a previous rule (`transformLinkAttributesToData`) already transformed the XLink Attributes represented as dataset-attributes in view layer to their `xlink:` pendants.

You may want to enforce this, adding "lowest" priority to this rule (it should run close to the sanitizer).

**Example Application**: To make the configuration better visible (as a demo), and also to better support evaluating this rule in manual tests, I would suggest adding it to our example application.